cli.py

```diff
@@ -44,6 +46,23 @@ def gen_postman_collection(filename='timetracker-api-postman-collection.json',
     save_data(parsed_json, filename)
 
 
+@cli_manager.command
```

Comment:
quick question: is there any chance to perform db/schema migrations by steps? 
If not, what we are going to do if we need to undo just the last change executed for example?

Reply:
This was not even requested in this issue. I just created these commands because of a need I had during my work with the database. 
I case something beyond a seeding is required we can tackle it down in another ticket. Flask migrates looks like a good choice at a glance.

Reply:
gotcha.

Reply:
https://github.com/ioet/time-tracker-api/issues/36